build.sbt

```diff
@@ -20,22 +20,33 @@ scalacOptions ++= Seq(
   "-Ywarn-numeric-widen" // Warn when numerics are widened.
 )
 
+// Added to get info about Java warnings (compiling with JDK 8u47) AKa270116
+//
+javacOptions ++= Seq(
+  "-Xlint:unchecked"
+)
+
 resolvers += Resolver.mavenLocal
 resolvers += "Maven Central Server" at "http://repo1.maven.org/maven2"
 
-libraryDependencies ++= Seq(
-  "com.typesafe.akka" %% "akka-actor" % "2.3.11",
-  "com.google.guava" % "guava" % "19.0",
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
-  "com.fasterxml.jackson.core" % "jackson-core" % "2.7.0",
-  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.6.3",
-  "ch.qos.logback" % "logback-classic" % "1.1.3",
-  "com.typesafe.akka" %% "akka-http-experimental" % "2.0.2",
-
-  "com.typesafe.akka" %% "akka-testkit" % "2.3.11" % "test",
-  "org.scalatest" %% "scalatest" % "2.2.4" % "test",
-  "io.undertow" % "undertow-core"    % "1.2.12.Final" % "test",
-  "io.undertow" % "undertow-servlet" % "1.2.12.Final" % "test",
-  "org.apache.commons" % "commons-io" % "1.3.2" % "test",
-  "com.google.code.findbugs" % "jsr305" % "1.3.9" % "test"
-)
+libraryDependencies ++= {
+  val akkaVer = "2.3.11"
+  val undertowVer = "1.2.12.Final"
+
+  Seq(
+    "com.typesafe.akka" %% "akka-actor" % akkaVer,
+    "com.google.guava" % "guava" % "19.0",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
+    "com.fasterxml.jackson.core" % "jackson-core" % "2.7.0",
+    "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.6.3",
+    "ch.qos.logback" % "logback-classic" % "1.1.3",
+    "com.typesafe.akka" %% "akka-http-experimental" % "2.0.2",
+
+    "com.typesafe.akka" %% "akka-testkit" % akkaVer % "test",
+    "org.scalatest" %% "scalatest" % "2.2.4" % "test",
+    "io.undertow" % "undertow-core"    % undertowVer % "test",
+    "io.undertow" % "undertow-servlet" % undertowVer % "test",
+    "org.apache.commons" % "commons-io" % "1.3.2" % "test",
+    "com.google.code.findbugs" % "jsr305" % "1.3.9" % "test"
+  )
```

Comment:
I like DRY. So placed the versions that are needed multiple times into a variable.